### ISSUE TITLE
Is it still active?

### ISSUE THREAD

**Issue by partizanos** [CamHenlin/roguish#4]
Is it still active?

**Comment by snelson3** [comment]
No. This was a class project 2-3 years ago and it hasn't been touched since. Do you have any questions about it?

**Comment by partizanos** [comment]
No i just checked the demo trying to see and didn't have to set it up.
Maybe you could add a github page with the game :)

**Comment by jeffhou** [comment]
@CamHenlin the links are either dead or redirect to malicious website.
